packages/view/backend/sirius-components-view-builder/src/main/java/org/eclipse/sirius/components/view/builder/generated/diagram/DiagramBuilders.java

```diff
@@ -359,5 +361,15 @@ public DropNodeToolBuilder newDropNodeTool() {
         return new DropNodeToolBuilder();
     }
 
+    /**
```

Comment:
Same comment as the one before

Reply:
I was mistaken, it has already been rebased on top of the commit I was thinking about. There's nothing to change here